### PR TITLE
Split libxprof into libxtimeline and libxtally

### PR DESCRIPTION
**Pull request by DonAurelio**
We split libxprof into two different components (btx_tally.c and btx_timeline.c). This is the preamble to migrate such components into metababel.

### CODE REVIEW

File: xprof/btx_tally.c [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#include <babeltrace2/babeltrace.h>
+#include "tally.h"
```
Comment (TApplencourt) [inline xprof/btx_tally.c:2]: Maybe rename it to `btx_tally` then?

File: xprof/btx_timeline.c [hunk 2]
```diff
@@ -1,21 +1,10 @@
 #include <babeltrace2/babeltrace.h>
-#include "tally.h"
 #include "timeline.h"
```
Comment (TApplencourt) [inline xprof/btx_timeline.c:2]: btx_timeline?

File: xprof/btx_tally.c (hunk 1 above)
Comment (DonAurelio) [reply xprof/btx_tally.c:2]: Makes sense, since we have babeltrace code spread on these files. I understand proper babeltrace stuff should be btx_.....

**Comment by TApplencourt** [comment]
LGTM!

**Review by TApplencourt — APPROVED** [review]

**Review by Kerilk — APPROVED** [review]
LGTM.

**Pull request merged**